elements/pf-alert/demo/custom-icons.html

```diff
@@ -1,5 +1,4 @@
-<link rel="stylesheet" href="demo.css">
-<script type="module" src="pf-alert.js"></script>
+<link rel="stylesheet" href="demo.css"><script type="module" src="pf-alert.js"></script>
```

Comment:
For the sake of clarity, What I mean by inline is to replace `<link href` with <style and <script src with <script

In other words, each demo file would be self contained except for the files we ship in our package